.github/workflows/integration-workflow.yml

```diff
@@ -89,10 +89,49 @@ jobs:
         node ./scripts/run-yarn.js build:plugin-version
         node ./scripts/run-yarn.js build:plugin-workspace-tools
 
+    - name: 'Build dist'
+      if: matrix.node == '12' && matrix.platform == 'ubuntu-latest'
+      run: |
+        ./scripts/build-dist.sh
+
+    - name: 'Upload dist'
+      uses: actions/upload-artifact@master
+      if: matrix.node == '12' && matrix.platform == 'ubuntu-latest'
+      with:
+        name: dist
+        path: artifacts
+
     - name: 'Run the integration tests'
       run: |
         node ./scripts/run-yarn.js test:integration --runInBand
 
     - name: 'Run the unit tests'
       run: |
         node ./scripts/run-yarn.js test:unit
+
+  build-deb:
+    needs: [acceptance]
+    name: 'Build Linux Packages'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/download-artifact@master
```

Comment:
The idea here was that instead of building Yarn multiple times, it's instead just built once, and other jobs retrieve the built artifacts. However, this also means that the Debian package build can't start until all the unit and integration tests finish running. I might just deal with it building multiple times.

Reply:
I think it's fine to rebuild it, as they are run in parallel anyway. Maybe later we could find a solution to only build it once for all the workflows (not only the jobs in one workflow).

Reply:
Cool, will do.